### PR TITLE
add py code to automate platform versions

### PR DESCRIPTION
**Pull request by mirnawong1**
adding some wizard code from @dbeatty10 (who called this out) to automate versioning

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `website/static/_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/docs/supported-data-platforms.md [hunk 1]
```diff
@@ -21,52 +21,52 @@ The following are **Verified adapters** ✓ you can connect to either in dbt Clo
...
+    body="<a href='/docs/cloud/connect-data-platform/connect-starburst-trino'><img src='/img/icons/dbt-bit.svg' width='7%'/>Set up in dbt Cloud </a> <br /><a href='/docs/core/connect-data-platform/trino-setup'><img src='/img/icons/dbt-bit.svg' width='7%'/>Install using the CLI  </a> <br /> <br /> <a href=https://badge.fury.io/py/dbt-trino><img src=https://badge.fury.io/py/dbt-trino.svg/>"
     icon="starburst"/>
 
 <Card
     title="Dremio*"
-    body="<a href='/docs/core/connect-data-platform/dremio-setup'><img src='/img/icons/dbt-bit.svg' width='7%'/>Install using the CLI  </a> <br /><br /> Latest version: 1.4<br /> 🚧 Verification in progress"
+    body="<a href='/docs/core/connect-data-platform/dremio-setup'><img src='/img/icons/dbt-bit.svg' width='7%'/>Install using the CLI  </a> <br /><br /> <a href=https://badge.fury.io/py/dbt-dremio><img src=https://badge.fury.io/py/dbt-dremio.svg/>br /> 🚧 Verification in progress"
```
Comment (mirnawong1) [inline website/docs/docs/supported-data-platforms.md]:
```suggestion
    body="<a href='/docs/core/connect-data-platform/dremio-setup'><img src='/img/icons/dbt-bit.svg' width='7%'/>Install using the CLI  </a> <br /><br /> <a href=https://badge.fury.io/py/dbt-dremio><img src=https://badge.fury.io/py/dbt-dremio.svg/><br /> 🚧 Verification in progress"
```

File: website/docs/docs/supported-data-platforms.md [hunk 2]
```diff
@@ -21,52 +21,52 @@ The following are **Verified adapters** ✓ you can connect to either in dbt Clo
...
     icon="snowflake"/>
 
 <Card
     title="Spark"
-    body="<a href='/docs/cloud/connect-data-platform/connect-apache-spark'><img src='/img/icons/dbt-bit.svg' width='7%'/>Set up in dbt Cloud </a> <br /><a href='/docs/core/connect-data-platform/spark-setup'><img src='/img/icons/dbt-bit.svg' width='7%'/>Install using the CLI </a> <br /><br /> Latest version: 1.4"
+    body="<a href='/docs/cloud/connect-data-platform/connect-apache-spark'><img src='/img/icons/dbt-bit.svg' width='7%'/>Set up in dbt Cloud </a> <br /><a href='/docs/core/connect-data-platform/spark-setup'><img src='/img/icons/dbt-bit.svg' width='7%'/>Install using the CLI </a> <br /><br /> <a href=https://badge.fury.io/py/dbt-spark><img src=https://badge.fury.io/py/dbt-spark.svg/></a>"
     icon="databricks"/>
 
 <Card
     title="Starburst"
```
Comment (dbeatty10) [inline website/docs/docs/supported-data-platforms.md:58]: Should this mention Trino at all?

Not blocking for this PR, but curious if we should update this page to include "Trino" somewhere.

**Review by dbeatty10 — APPROVED** [review]
👍 Look at those auto-updating version numbers! 🤩

image

File: website/docs/docs/supported-data-platforms.md (hunk 2 above)
Comment (mirnawong1) [reply website/docs/docs/supported-data-platforms.md:58]: good catch! opening another pr for this!

**Pull request merged**